portability-api/src/main/java/org/datatransferproject/api/action/transfer/CreateTransferJobAction.java

```diff
@@ -79,9 +80,14 @@ public TransferJob handle(CreateTransferJob request) {
     String encodedSessionKey = BaseEncoding.base64Url().encode(sessionKey.getEncoded());
 
     String encryptionScheme = request.getEncryptionScheme();
-    PortabilityJob job =
-        createJob(encodedSessionKey, dataType, exportService, importService, exportInformation, encryptionScheme);
-
+    PortabilityJob job;
+    try {
+      job =
+          createJob(encodedSessionKey, dataType, exportService, importService, exportInformation,
+              encryptionScheme);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
```

Comment:
Should some additional information be logged, e.g. the request?

Reply:
The request could contain PII so I dont want to unnecessarily log it. And in this case the exception should be enough (it will contain the reason the deserialization failed).